src/sentry/issues/endpoints/organization_derive_code_mappings.py

```diff
@@ -47,14 +47,17 @@ def get(self, request: Request, organization: Organization) -> Response:
         :param string stacktraceFilename:
         :auth: required
         """
+        # XXX: Evaluate what else does the UI send us
```

Comment:
I have added a task so I don't forget to check if "Set up code mappings" works for Java.